CHANGELOG.md

```diff
@@ -4,6 +4,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.12.0] - 2024-04-15
+
+### Known Issues:
+
+- Default Kconfig values may not propagate correctly if the Golioth SDK
+  is not listed first in the West manifest. To workaround this, ensure
+  that Golioth is listed first in your application's manifest.
+
+### Added:
+
+- Measure code coverage in continuous integration tests
+- Add socket hello_nrf91_offload Zephyr example to demonstrate DTLS
+  sockets offload.
+- Zephyr: add rak5010_nrf52840 support to all examples
+- ESP-IDF: add fw_update example to demonstrate OTA firmware update
+
+### Changed:
+
+- Zephyr: example code now selects necessary Kconfig symbols explicitly
```

Comment:
This isn't accurate. The examples still use the defconfig files, but the defconfig is included explicitly by each example instead of implicitly included via the SDK. What this means is that _client_ applications can no longer use the defconfig files.
https://github.com/golioth/golioth-firmware-sdk/pull/420

Reply:
I've updated this entry as follows:

```
- Zephyr: example code now explicitly sources Kconfig.defconfig file to 
  select necessary Kconfig symbols from the example common files.
```